tools/dart/create_updated_flutter_deps.py

```diff
@@ -95,15 +93,38 @@ def Main(argv):
         '  # The lines between blank lines above and below are generated by a script. See create_updated_flutter_deps.py\n'])
       while i < len(lines) and len(lines[i].strip()) > 0:
         i = i + 1
-      for (k, v) in sorted(updated_deps.iteritems()):
+      for (k, v) in sorted(updated_vars.iteritems()):
         updatedfile.write("  '%s': '%s',\n" % (k, v))
       updatedfile.write('\n')
-    i = i + 1
 
-  if len(missing_deps) > 0:
-    print("Error: flutter dependencies %s removed from dart" % missing_deps)
-    print("Intermediate results are in %s" % updatedfilename)
-    return 1
+    elif lines[i].startswith("  # WARNING: Unused Dart dependencies"):
+      updatedfile.write('\n')
+      i = i + 1
+      while i < len(lines) and (lines[i].startswith("  # WARNING: end of dart dependencies") == 0):
+        i = i + 1
+      for (k, v) in sorted(old_deps.iteritems()):
+        if (k.startswith('src/third_party/dart/')):
+          for (dart_k, dart_v) in (new_deps.iteritems()):
+            dart_k_suffix = dart_k[len('sdk/') if dart_k.startswith('sdk/') else 0:]
+            if (k.endswith(dart_k_suffix)):
+              updated_value = dart_v
```

Comment:
Does this do anything? We overwrite `updated_value` in the next line.

Reply:
Ah, right!